examples/elf/Makefile

```diff
@@ -30,6 +30,8 @@ CSRCS = cromfs.c
 endif
 CSRCS += dirlist.c
 CSRCS += symtab.c
+tests$(DELIM)symtab.c_CFLAGS = -fno-builtin
```

Comment:
should we expose -fno-builtin with a macro in Toolchain.defs/Make.defs to avoid the hardcode value in the common file?

Reply:
ideally, maybe.
we are using `-fno-lto` in the common file as well.